libp2p/protocols/kademlia/kademlia.nim

```diff
@@ -182,6 +238,21 @@ proc new*(
 
           # Peer is useful. adding to rtable
           discard kad.rtable.insert(conn.peerId)
+        of MessageType.putValue:
+          var record = msg.record.get()
```

Comment:
Let's use valueOr, unless you're 100% sure a msg will contain a record. Same in lines 243 and 244

Reply:
same as https://github.com/vacp2p/nim-libp2p/pull/1582#discussion_r2242685948 (set up a TODO)

[08c994b](https://github.com/vacp2p/nim-libp2p/pull/1582/commits/08c994b32af9064ffdcdb19fff8c0fbce7dc8ed7)